com.sweetscape.ZeroOneZeroEditor.yml

```diff
@@ -0,0 +1,58 @@
+app-id: com.sweetscape.ZeroOneZeroEditor
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: 010editor
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=home
+
+modules:
+  - name: perl-libs
+    buildsystem: simple
+    build-commands:
+      - 'perl-libs/install.sh'
+    post-install:
+      - 'chmod -Rf u+w /app/lib/perl5/site_perl'
+    sources:
+      - generated-sources.json
+
+  - name: unpack-install-jammer
+    buildsystem: simple
+    build-commands:
+      - chmod +x extract.pl
+      - mkdir -p /app/bin
+      - mv extract.pl /app/bin/extract.pl
+    sources:
+      - type: git
+        url: https://github.com/lod/unpack-install-jammer.git
+        tag: 0.2.0
+        commit: 41faa5a8c7925db4ff7ca665cca478979d177de4
+
+  - name: 010editor
+    only-arches:
+      - x86_64
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true # Fix error while loading shared libraries: libQt5Core.so.5: object file has no loadable segments
+    build-commands:
+      - perl -I /app/lib/perl5/site_perl/5.34.0/ /app/bin/extract.pl --quiet 010EditorLinux64Installer
+      - mv install_dir/$HOME/010editor /app/lib/010editor
+      - chmod -x /app/lib/010editor/Data/010Repository.zip
+      - mkdir -p /app/bin 
+      - ln -s /app/lib/010editor/010editor /app/bin/010editor
+      - mkdir -p /app/share/icons/hicolor/128x128/apps/
+      - cp /app/lib/010editor/010_icon_128x128.png /app/share/icons/hicolor/128x128/apps/com.sweetscape.ZeroOneZeroEditor.png
+      - install -Dm0644 com.sweetscape.ZeroOneZeroEditor.desktop /app/share/applications/com.sweetscape.ZeroOneZeroEditor.desktop
+      - install -Dm0644 com.sweetscape.ZeroOneZeroEditor.appdata.xml /app/share/appdata/com.sweetscape.ZeroOneZeroEditor.appdata.xml
+    sources:
+      - type: archive
+        url: https://download.sweetscape.com/010EditorLinux64Installer12.0.1.tar.gz
```

Comment:
Please configure https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
Done https://github.com/flathub/com.sweetscape.ZeroOneZeroEditor/commit/2be87b63f169f7da39410f740aa911dc6f6485c2